cmd/tempo-query/tempo/plugin.go

```diff
@@ -59,6 +59,9 @@ func (b *Backend) GetTrace(ctx context.Context, traceID jaeger.TraceID) (*jaeger
 		req.Header.Set(user.OrgIDHeaderName, tenantID)
 	}
 
+	// Set content type to grpc
+	req.Header.Set("Accepts", "application/protobuf")
```

Comment:
Let's use the standard header `Accept`:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Probably would be wise to define this in one place and reference it everywhere we need it.  Perhaps a new file: `util/http/headers.go` ?  I'm also surprised to see I'm not finding this defined in the go std lib.

Reply:
Agreed, added separate util file `http.go`.